lib/galaxy/datatypes/proteomics.py

```diff
@@ -72,7 +72,7 @@ def display_peek(self, dataset):
 class ProtXmlReport(Tabular):
     """protxml converted to tabular report"""
     edam_data = "data_2536"
-    file_ext = "tsv"
+    file_ext = "protxml.tsv"
```

Comment:
I saw you make this comment the other day, and it seemed wrong at the time - that this extension needs to match the datatypes_conf.xml file. I thought ``file_ext`` in this file is used for downloads - and that many things with the same ``file_ext`` could all have different ``datatypes_conf.xml`` keys.

I wouldn't make this change - do you have evidence that it needs to match up?